docs/xostor/xostor.md

```diff
@@ -852,3 +852,64 @@ Note: iptables config must also be modified to remove LINSTOR port rules (edit `
 For more info:
 - https://linbit.com/blog/linstors-auto-evict/
 - https://linbit.com/drbd-user-guide/linstor-guide-1_0-en/#s-linstor-auto-evict
+
+### How to-enable dm-cache?
+
+:::warning
+This feature is currently experimental and is not covered by XCP-ng support.
+:::
+
+On each host, create a new PV and VG using your cache devices:
+```
+pvcreate linstor_group_cache <CACHE_DEVICES>
+vgcreate linstor_group_cache <CACHE_DEVICES>
+```
+
+Using the controller, enable the cache:
+```
+# Make sure the primary resource group is configured with cache support.
```

Comment:
IDK if it's on purpose, but in general I prefer to put all the comment text in regular text to only leave the command to execute in a code block. I find it less confusing on what to copy/paste (at the risk of reading less what's inside). No strong opinion on this, just an input :)